src/echo.js

```diff
@@ -23,12 +27,23 @@ function _echo(opts, messages) {
   // allow strings starting with '-', see issue #20
   messages = [].slice.call(arguments, opts ? 0 : 1);
 
-  if (messages[0] === '-e') {
-    // ignore -e
+  var option = ['-e', '-n', '-ne', '-en'].indexOf(messages[0]);
+  var output;
+
+  if (option >= 0) {
```

Comment:
Yeah, but it won't throw the error as an exception, it'll return early from the `echo` function. You can hack around it by setting `config.fatal` to have it throw an exception. A clearer way is to add an option to `common.error` to throw an exception instead of exiting (regardless of `config.fatal`).

The approach you have here works if we only have two options, but it gets cumbersome if we ever support `-E` (which we may want to support in the future).